actions/helm-deploy/entrypoint.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/sh
+
+set -eux
+
+if [ -f helmprerundone ]; then
+  printf "Skipping helmprerun\n"
+else
+  helm plugin install https://github.com/thomastaylor312/helm-namespace
```

Comment:
Do we still need this chunk of code? Shouldn't the dockerfile now install this?

Reply:
You're totally right! I'll take that out

Reply:
It turns out this is what made the previous test pass 😞 After I took this out the [action](https://github.com/DataBiosphere/jade-data-repo/runs/2187432234?check_suite_focus=true) failed with `Error: unknown command "namespace" for "helm"`.